chat-widget/src/components/webchatcontainerstateful/WebChatContainerStateful.tsx

```diff
@@ -203,8 +203,12 @@ export const WebChatContainerStateful = (props: ILiveChatWidgetProps) => {
         .webchat__bubble:not(.webchat__bubble--from-user) .webchat__bubble__content {
             border-radius: 0 !important; /* Override border-radius */
         }
+
+        .webchat__stacked-layout_container>div {
+            background: ${(props?.webChatContainerProps?.containerStyles as IRawStyle)?.background?? ""}
```

Comment:
when the props is empty, this will override any style, do we need to define a default style ?, in that case when props is not passed, then it will use default

Reply:
Leaving it empty because it will be overriden by the bot framework default background color.